### ISSUE TITLE
brightness_move for groups

### ISSUE THREAD

**Issue by eilmi** [Koenkk/zigbee2mqtt#2620]
Hi Koen,
I just updated to zigbee2mqtt 1.8.0 and I was wondering if it´s possible to implement the new command "brightness_move" also in groups because right now it`s only working for single devices
(tried it with Ikea Gu10 spots)

**Comment by Koenkk** [comment]
Added!

**Comment by eilmi** [comment]
Thank you!

**Comment by sergiigladchuk** [comment]
Hi all,
I just tested this `brightness_move` command for my grouped lights. All works fine, the only problem is when i send command `{"brightness_move" : "stop"}`  the Z2M is not publishing state message with current brightness for the group. I have checked it for single light and message is being publish in this case. I use latest master branch.
@Koenkk is this by design or impossible to implement for the groups, or I need some special setup in my groups.yaml?
